go.mod

```diff
@@ -2,11 +2,12 @@ module github.com/fastly/terraform-provider-fastly
 
 go 1.14
 
+replace github.com/fastly/go-fastly/v3 v3.0.0 => github.com/opencredo/go-fastly/v3 v3.0.0-20210208131624-9ccb395d6808
```

Comment:
Temporary redirect to OC fork until https://github.com/fastly/go-fastly/pull/259 is merged